charts/prometheus-nginx-exporter/Chart.yaml

```
@@ -2,7 +2,7 @@
 apiVersion: v2
 description: A Helm chart for NGINX Prometheus Exporter
 name: prometheus-nginx-exporter
-version: 1.4.3
+version: 1.4.4
```

This change is a new feature.

```suggestion
version: 1.5.0
```

---

Done, changed.